surveyo/README.md

```diff
@@ -32,15 +32,34 @@ Surveyo is a sample app that provides users with a survey tool that they can use
 
 ### Back-end
 
-- [Slash GraphQL](https://dgraph.io/slash-graphql)—a fully managed GraphQL backend service
+- [Slash GraphQL](https://dgraph.io/slash-graphql)—a fully managed GraphQL backend service.
```

Comment:
Let's change Slash GraphQL to Dgraph Cloud or something related. We are not using the name "Slash GraphQL" it was the previous name of our cloud.

Reply:
Done.